gulpfile.js

```diff
@@ -124,7 +119,7 @@ gulp.task('clean', getTask('./gulp/clean', {
 	pkgName: pkg.name
 }));
 
-gulp.task('pre-commit', gulp.series('lint', 'check-epam-email', 'check-deps-exact'));
+gulp.task('pre-commit', gulp.series('check-epam-email', 'check-deps-exact'));
```

Comment:
Let's also remove epam-email checker in scope of this PR, redundant check for now